triton/kubernetes/FIL/interact_with_triton.ipynb

```diff
@@ -0,0 +1,272 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[0.72424215, 0.27575785],\n",
+       "       [0.760525  , 0.239475  ],\n",
+       "       [0.793439  , 0.20656103],\n",
+       "       ...,\n",
+       "       [0.33843   , 0.66157   ],\n",
+       "       [0.47233915, 0.52766085],\n",
+       "       [0.93259996, 0.06740003]], dtype=float32)"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result_http = request_http.as_numpy('output__0')\n",
+    "result_http"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### gRPC Client Example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Is Server Ready: True\n",
+      "Is FIL model ready: True\n"
+     ]
+    }
+   ],
+   "source": [
+    "grpc_client = triton_grpc.InferenceServerClient(\n",
+    "    url=f'{host}:{port}',\n",
+    "    verbose = False\n",
+    ")\n",
+    "\n",
+    "while (not (grpc_client.is_server_ready() or grpc_client.is_model_ready('fil'))):\n",
+    "    print(\"Waiting on server ready\")\n",
+    "    time.sleep(5)\n",
+    "    \n",
+    "print(f\"Is Server Ready: {grpc_client.is_server_ready()}\")\n",
+    "print(f\"Is FIL model ready: {grpc_client.is_model_ready('fil')}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = numpy.random.rand(samples, features).astype('float32')\n",
+    "triton_input_grpc = triton_grpc.InferInput(\n",
+    "    'input__0',\n",
+    "    [samples, features],\n",
+    "    'FP32'\n",
+    ")\n",
+    "\n",
+    "triton_input_grpc.set_data_from_numpy(data)\n",
+    "triton_output_grpc = triton_grpc.InferRequestedOutput('output__0')\n",
+    "\n",
+    "request_grpc = grpc_client.infer(\n",
+    "    'fil',\n",
+    "    model_version='1',\n",
+    "    inputs=[triton_input_grpc],\n",
+    "    outputs=[triton_output_grpc]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[0.72424215, 0.27575785],\n",
+       "       [0.760525  , 0.239475  ],\n",
+       "       [0.793439  , 0.20656103],\n",
+       "       ...,\n",
+       "       [0.33843   , 0.66157   ],\n",
+       "       [0.47233915, 0.52766085],\n",
+       "       [0.93259996, 0.06740003]], dtype=float32)"
+      ]
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result_grpc = request_grpc.as_numpy('output__0')\n",
+    "result_grpc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Compare Results"
```

Comment:
It might be interesting to add a bit of benchmark result so the user can understand the benefits of GPU when serving XGBoost through Triton?

Reply:
Great point Dong, will add additional content after verifying auto-scaling